### PR TITLE
Add AltTab to the window management section

### LINKED ISSUE #799: Add AltTab
NOTE: Each discrete (stand-alone) request should be in its own issue.

**Put "Add ____" as Title**
 [x] Done

**Provide a link to the proposed addition.**
https://github.com/lwouis/alt-tab-macos

**Explain why it should be added**
It's just makes Macos app switch usable, especially when you have many spaces

**Additional context**
I am not the author of this project, just a fan. Other info are available to the provided link.

### Issue Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the issue. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before responding to the issue.

- [x] I have checked for other similar issues
- [x] I have explained why this change is important
- [x] I have added necessary documentation (if appropriate)

### PR DESCRIPTION
**Pull request by mnin**
Close #799

### Types of Changes

**What types of changes does your PR introduce?** Put an x in all boxes that apply

- [x] New addition to list
- [ ] Fixing bug in existing item in list
- [ ] Removing item from list
- [ ] Changing structure (organization) of list

### Proposed Changes

Add another window switcher AltTab to the list. Because AltTab is currently being maintained and is a good open source alternative.

### PR Checklist

Put an x in the boxes once you've completed each step. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.

- [x] I have read the CONTRIBUTING guide
- [x] I have explained why this PR is important
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -837,6 +837,7 @@ Any comments, suggestions? [Let us know!](https://github.com/jaywcjlove/awesome-
 
 #### Window Management
 
+* [AltTab](https://alt-tab-macos.netlify.app) - Open source window switcher with window previews. [![Open-Source Software][OSS Icon]
```
Comment (jaywcjlove) [inline README.md:840]: Open-Source URL: https://github.com/lwouis/alt-tab-macos @mnin

Comment (mnin) [reply README.md:840]: @jaywcjlove would you like add both URLs here? Thought, one should be enough and the given explain it better than the GitHub project URL.

Comment (mnin) [reply README.md]: @jaywcjlove done

Comment (jaywcjlove) [reply README.md]: @mnin thx!

**Pull request merged**